### ISSUE TITLE
Documentation Issue: Missing data.js File in Build Instructions

### ISSUE THREAD

**Issue by tristdrum** [commenthol/date-holidays#467]
Hello `date-holidays` maintainers,

I hope you're doing well. I've recently attempted to contribute to the `date-holidays` project by following the instructions outlined in the `CONTRIBUTING.md` file. However, I encountered a build issue due to a missing `data.js` file that seems to be referenced in the project.

#### Issue Description:
During the build process, after running `npm run build`, Rollup throws an error indicating it could not resolve "./data.js" from "src/Holidays.js". Upon further inspection, I realized there is no mention of the `data.js` file in the contributing guidelines, nor does such a file exist in the repository.

#### Error Message:
```
[!] RollupError: Could not resolve "./data.js" from "src/Holidays.js"
```

#### Expected Behavior:
The build process completes without errors, allowing contributors to test their changes locally before submitting a pull request.

#### Suggested Actions:
- **Clarification in `CONTRIBUTING.md`**: If the `data.js` file is generated during an earlier step or needs to be created manually, could this process be detailed in the contributing guidelines?
- **Missing File**: If `data.js` is a file that should exist within the repository but is inadvertently omitted, it would be helpful to include it or provide the necessary steps to generate it.

I believe that addressing this issue will make the contributing process smoother for future contributors. Thank you for your attention to this matter and for maintaining such a valuable project. I look forward to your guidance on how to proceed.

Best regards,
@tristdrum

**Comment by tristdrum** [comment]
Hi @commenthol!

I've just submitted my [second PR](https://github.com/commenthol/date-holidays/pull/468) (and hopefully learnt my lessons from the [first one](https://github.com/commenthol/date-holidays/pull/447)) but cannot test locally because of this issue. 🙈 

Thank you 🙏

**Comment by commenthol** [comment]
@tristdrum 
You are right. The first time the data.js file is not there. Try running `npm run yaml` before `npm run build` that's it.
Will take care on a PR which solves this.

**Comment by tristdrum** [comment]
@commenthol thank you for the help, and the tireless contributions!

I appreciate you!
